set_version.py

```diff
@@ -5,7 +5,7 @@
 
 import os
 
-major_minor = "1.0"
+major_minor = "1.1"
```

Comment:
I will look into baking the version into pipeline variables so that we don't need a code change when we release.